lib/internal/process.js

```diff
@@ -70,10 +70,48 @@ function setupConfig(_source) {
 
 function setupKillAndExit() {
 
-  process.exit = function(code) {
+  const kExitTimer = Symbol('kExitTimer');
```

Comment:
I like the little `k`. It communicates more than just something not changing; It says “I’m an opaque constant, I’m an identifier for some thingy, and my exact value is irrelevant.”.